import java.lang.Exception
import java.util.*

class Main {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            println("start")
            var result = Solution().solution(5, 12)
            println("${result}")
        }

    }
}
